packages/api/db/migration/20230901204048_create_general_sale_table.js

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (c) 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export const up = async function (knex) {
```

Comment:
I think I was picturing using the existing 'sale' table but add the value and note columns. In this way it would mirror the 'farmExpense' table. For crop sales -- value would be blank or a sum total.

Do think we need a new table?

Reply:
![Screenshot 2023-09-12 at 3 36 14 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/33141219/9f00be1e-b7e9-47c0-85e6-c54b5fd56783)
I'm fine with adding columns to the sale table. It seemed that we were expecting other types that have unique fields, so I thought there would be one table for each form.